Repairs/app/src/main/res/layout/activity_equipment_2.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I am not sure what the difference between this and activity_equipment_1 is? May be better to just have the one equipment page